ext/standard/basic_functions.stub.php

```diff
@@ -80,3 +80,11 @@ function closelog (): bool {}
 
 function syslog (int $priority, string $message): bool {}
 #endif
+
+#ifdef HAVE_INET_NTOP
+function inet_ntop (string $in_addr): string {}
```

Comment:
`inet_ntop` may also return `false`, so
```suggestion
/** @return string|false */
function inet_ntop (string $in_addr) {}
```